funnel/models/proposal.py

```diff
@@ -248,6 +248,14 @@ def move_to(self, project):
         self.url_id = None
         self.make_id()
 
+    @with_roles(call={'admin'})
+    def transfer_to(self, user):
+        """
+        Transfer the proposal to a new user and speaker
+        """
+        self.user = user
+        self.speaker = user
```

Comment:
https://github.com/hasgeek/funnel/issues/456

Reply:
There is a workflow issue here somewhere. Because if user/creator and speaker are different, frontend shows a `via <creator>` part after the speaker name. Which is unwanted by the media team. Maybe we need to talk to them once about their thought on this.

Reply:
Which is why I always have to update both `user` and `speaker` whenever they ask me to transfer a proposal.

Reply:
They don't have a choice in this matter. We're preserving the identity of the proposal's creator.

Reply:
Once changesets are implemented (#395), we will even publicly display _when_ the proposal was transferred.